### PR TITLE
=htc #482 consistent port names logged in Graph stages

### PR DESCRIPTION
**Pull request by gosubpl**
the Inlet and Outlet ports were renamed in Graph stages to be more consistent. Names used for logging now also include class name to help with diagnostics based on logs only.

### CODE REVIEW

**Comment by gosubpl** [comment]
Refs #482

**Comment by gosubpl** [comment]
Please let me know how do you like it. Especially the plural in the names. Seems natural when you construct the port but maybe less intuitive when used in `pull` or `push`.

File: akka-http-core/src/main/scala/akka/http/impl/engine/parsing/BodyPartParser.scala [hunk 1]
```diff
@@ -62,10 +62,10 @@ private[http] final class BodyPartParser(
     if (illegalHeaderWarnings) log.warning(errorInfo.withSummaryPrepended("Illegal multipart header").formatPretty)
   }
 
-  val in = Inlet[ByteString]("BodyPartParser.in")
```
Comment (ktoso) [inline akka-http-core/src/main/scala/akka/http/impl/engine/parsing/BodyPartParser.scala]: those are fine as is IMO. No need to go all crazy with those renames, as long as we can identify which port and class it serves it's purpose

File: akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpRequestParser.scala [hunk 2]
```diff
@@ -32,10 +32,10 @@ private[http] final class HttpRequestParser(
   import HttpMessageParser._
   import settings._
 
-  val in = Inlet[SessionBytes]("HttpRequestParser.in")
-  val out = Outlet[RequestOutput]("HttpRequestParser.out")
```
Comment (ktoso) [inline akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpRequestParser.scala]: let's keep in/out in classes where we have simple Flows - it's simpler, no need to add the full types into the names

File: akka-http-core/src/main/scala/akka/http/impl/util/One2OneBidiFlow.scala [hunk 3]
```diff
@@ -44,13 +44,13 @@ private[http] object One2OneBidiFlow {
    *    +--------------------+
    */
   class One2OneBidi[I, O](maxPending: Int) extends GraphStage[BidiShape[I, I, O, O]] {
-    val in = Inlet[I]("One2OneBidi.in")
-    val out = Outlet[O]("One2OneBidi.out")
-    val toWrapped = Outlet[I]("One2OneBidi.toWrapped")
-    val fromWrapped = Inlet[O]("One2OneBidi.fromWrapped")
+    val in = Inlet[I]("One2OneBidi.In")
```
Comment (ktoso) [inline akka-http-core/src/main/scala/akka/http/impl/util/One2OneBidiFlow.scala]: keep as it was, lower case `in`

**Comment by gosubpl** [comment]
Thanks @ktoso for chiming in. What are your feelings around singular vs plural debate?

**Comment by ktoso** [comment]
Singualr please :)

**Comment by gosubpl** [comment]
One more question: in some places instead of hardcoded name we have `Logging.simpleName(this)`. Does it hurt us in any way? Leave as is?

File: akka-http-core/src/main/scala/akka/http/impl/engine/parsing/BodyPartParser.scala (hunk 1 above)
Comment (gosubpl) [inline akka-http-core/src/main/scala/akka/http/impl/engine/parsing/BodyPartParser.scala]: Done

File: akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpRequestParser.scala (hunk 2 above)
Comment (gosubpl) [inline akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpRequestParser.scala]: Done

File: akka-http-core/src/main/scala/akka/http/impl/engine/rendering/BodyPartRenderer.scala [hunk 4]
```diff
@@ -32,9 +32,9 @@ private[http] object BodyPartRenderer {
     new GraphStage[FlowShape[Multipart.BodyPart, Source[ChunkStreamPart, Any]]] {
       var firstBoundaryRendered = false
 
-      val in: Inlet[Multipart.BodyPart] = Inlet("BodyPartRenderer.in")
-      val out: Outlet[Source[ChunkStreamPart, Any]] = Outlet("BodyPartRenderer.out")
-      override val shape: FlowShape[Multipart.BodyPart, Source[ChunkStreamPart, Any]] = FlowShape(in, out)
+      val bodyPartsIn: Inlet[Multipart.BodyPart] = Inlet("BodyPartRenderer.bodyPartsIn")
```
Comment (gosubpl) [inline akka-http-core/src/main/scala/akka/http/impl/engine/rendering/BodyPartRenderer.scala]: Done

**Comment by gosubpl** [comment]
@ktoso should be better now. I have reverted some _overdoing_ :) Also moved plural -> singular.

**Comment by gosubpl** [comment]
@ktoso thanks again :)

**Comment by ktoso** [comment]
All good now, thanks!

**Comment by ktoso** [comment]
Refs #482

**Pull request merged**